AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Models/PackageManifestBuilderTests.swift

```diff
@@ -12,7 +12,6 @@ class PackageManifestBuilderTests: XCTestCase {
 
     let expected = """
 <contents of prefix>
-
```

Comment:
Why is this test expectation changed?  This feature doesn't involve the manifest.

Reply:
The test was failing; I don't think this test was being run as part of CI with the package refactor task from before.